### PR TITLE
Document set operations

### PR DESCRIPTION
**Pull request by sigilante**
What do you think?  This could work as a landing page for `++in` or something too.

### CODE REVIEW

**Comment by matthew-levan** [comment]
Excellent illustrations.

File: content/guides/additional/sets.md [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+```
+
+##  Set Relations
+
+First we consider the elementary operations between two sets.
+
+### Union ($A \cup B$)
+
+$$
+A \cup B \equiv \{ x : x \in A \;\textrm{or}\; x \in B \}
```
Comment (tinnus-napbus) [inline content/guides/additional/sets.md:107]: this syntax doesn't render properly in all cases

File: content/guides/additional/sets.md [hunk 2]
```diff
@@ -0,0 +1,230 @@
...
+```hoon
+> `(set @tas)`(silt `(list @tas)`~[%a %b %c %a])
+{%b %a %c}
+```
+
+### Add Members
+
+![](https://media.urbit.org/docs/hoon-syntax/set-addition.png)
+
+[`++put:in`](https://developers.urbit.org/reference/hoon/stdlib/2h#putin) adds an element $x$ to a set $A$.
```
Comment (tinnus-napbus) [inline content/guides/additional/sets.md:30]: all cases of the $something$ syntax render as-is. Is that meant to do something different?

**Review by tinnus-napbus — CHANGES_REQUESTED** [review]
Cool diagrams, looks good apart from my comments - is that syntax meant to invoke LaTeX or something? It seems like it's not working, at least not in the preview

**Comment by sigilante** [comment]
It's LaTeX, but I think we have to use a different invocation on the site come to think of it.

**Comment by sigilante** [comment]
OK, it's ready with regard to typography.

**Review by tinnus-napbus — APPROVED** [review]

**Comment by tinnus-napbus** [comment]
redone for new docs site in https://github.com/urbit/docs.urbit.org/pull/32

**Pull request closed without merging**